src/Zonotope.jl

```diff
@@ -684,3 +685,36 @@ function translate(Z::Zonotope{N}, v::AbstractVector{N}; share::Bool=false
     generators = share ? Z.generators : copy(Z.generators)
     return Zonotope(c, generators)
 end
+
+"""
+    overapproximate(Z::Zonotope)
```

Comment:
`overapproximate` functions should go in `Approximations.overapproximate.jl`. It should also take a second argument. But see the next comment.

Reply:
```suggestion
    overapproximate(Z::AbstractZonotope)
```
This make the function also applicable for other set types such as `Hyperrectangle`s.

Reply:
Hmm, it could be, but im not sure. This method is meant to be applied to overapproximation of zonotopes with parallelotopes (or polytopes in a later generalization), as in the referred article.

Reply:
Which comment are you referring to?